backend/notenverwaltung/src/main/java/ch/lalumamesh/notenverwaltung/repository/user/UserRepository.java

```diff
@@ -0,0 +1,8 @@
+package ch.lalumamesh.notenverwaltung.repository.user;
+
+import ch.lalumamesh.notenverwaltung.model.user.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+    User findByUsername(String username);
```

Comment:
Would be better to either user 

```findByUsernameOrThrow(String username)```

or 

```findByUsernameOrNull(String username)``` (this needs to be implemented first if I remember correctly)

Reply:
thats a koltin thing